src/wp-includes/class-wp-comment-query.php

```diff
@@ -1036,13 +1036,19 @@ protected function fill_descendants( $comments ) {
 			$child_ids           = array();
 			$uncached_parent_ids = array();
 			$_parent_ids         = $levels[ $level ];
-			foreach ( $_parent_ids as $parent_id ) {
-				$cache_key        = "get_comment_child_ids:$parent_id:$key:$last_changed";
-				$parent_child_ids = wp_cache_get( $cache_key, 'comment' );
-				if ( false !== $parent_child_ids ) {
-					$child_ids = array_merge( $child_ids, $parent_child_ids );
-				} else {
-					$uncached_parent_ids[] = $parent_id;
+			if ( $_parent_ids ) {
```

Comment:
Thanks for checking, as it will not load code for posts that do not have comments.